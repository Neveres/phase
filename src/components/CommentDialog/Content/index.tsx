export { default, default as Content } from './Content'
