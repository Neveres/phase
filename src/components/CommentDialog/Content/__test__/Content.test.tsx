import React from 'react'
import { create } from 'react-test-renderer'
import Content from '../Content'

describe('Content', () => {
  test('should render well uuid existed', () => {
    expect(
      create(
        <Content
          uuid="uuid"
          comments={[
            {
              name: 'name',
              message: 'message',
              postTime: 'postTime',
            },
            {
              name: 'name',
              message: 'message',
              postTime: 'postTime',
            },
            {
              name: 'anotherName',
              message: 'message',
              postTime: 'postTime',
            },
          ]}
          inputRef={null as any}
        />,
      ).toJSON(),
    ).toMatchSnapshot()
  })

  test('should render well uuid not existed', () => {
    expect(
      create(<Content uuid="" comments={[]} inputRef={null as any} />).toJSON(),
    ).toMatchSnapshot()
  })
})
