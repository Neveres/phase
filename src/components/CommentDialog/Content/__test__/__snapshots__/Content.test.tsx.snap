// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render well uuid existed 1`] = `
Array [
  <div
    className="css-s8pplc-Content"
  >
    <div>
      <div
        className="comment-header"
      >
        <div
          className="comment-name"
        >
          name
        </div>
        <div
          className="comment-post-time"
        >
          postTime
        </div>
      </div>
      <div
        className="comment-message"
      >
        message
      </div>
    </div>
    <div>
      <div
        className="comment-message"
      >
        message
      </div>
    </div>
    <div>
      <div
        className="comment-header"
      >
        <div
          className="comment-name"
        >
          anotherName
        </div>
        <div
          className="comment-post-time"
        >
          postTime
        </div>
      </div>
      <div
        className="comment-message"
      >
        message
      </div>
    </div>
  </div>,
  <input
    className="ant-input"
    data-testid="message-input"
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    placeholder="Leave your comment"
    type="text"
    value=""
  />,
]
`;

exports[`Content should render well uuid not existed 1`] = `
Array [
  "",
  <input
    className="ant-input"
    data-testid="message-input"
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    placeholder="Leave your comment"
    type="text"
    value=""
  />,
]
`;
